pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -2282,7 +2284,7 @@ ASTInsertIntoClause InsertIntoClause() :
 ASTValuesClause ValuesClause() :
 {}
 {
-    <VALUES> "(" ( Expression() | <_DEFAULT> ) ( "," ( Expression() | <_DEFAULT> ) )* ")"
+    <VALUES> [ "(" ( Expression() | <_DEFAULT> ) ( "," ( Expression() | <_DEFAULT> ) )* ")" | <IDENTIFIER> ]
```

Comment:
Is this correct syntax? Using a single identifier instead of the parenthesis seems not be described here: https://docs.oracle.com/en/database/oracle/oracle-database/18/sqlrf/INSERT.html#GUID-903F8043-0254-4EE9-ACC1-CB8AC0AF3423__I2122346 for the values clause.

Reply:
This code compile and works but is not documented in the Oracle syntax, maybe is not a best practice, but is not an error:

DECLARE
   l_employee   omag_employees%ROWTYPE;
BEGIN
   l_employee.employee_id := 500;
   l_employee.last_name := ‘Mondrian’;
   l_employee.salary := 2000;

   INSERT
     INTO omag_employees 
   VALUES l_employee;
END;
/

https://blogs.oracle.com/oraclemagazine/working-with-records

Reply:
This is actually a PLSQL extension:
https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/INSERT-statement-extension.html#GUID-D81224C4-06DE-4635-A850-41D29D4A8E1B